HandHistories.Objects/Hand/RunItTwice.cs

```diff
@@ -12,13 +12,124 @@ public sealed class RunItTwice
         /// <summary>
         /// The second board
         /// </summary>
-        public BoardCards Board;
+        public BoardCards Board = BoardCards.FromCards(String.Empty);
```

Comment:
Please keep it initalized to null. This will be an extra unnessecary allocation every time RunItTwice is initalized.

Reply:
Done.